docs/api-testing/on-prem/bloodhound.md

```diff
@@ -5,17 +5,19 @@ sidebar_label: Introduction
 description: "Bloodhound is an HTTP API Microgateway that acts are a reverse proxy."
```

Comment:
Description: "Bloodhound is an HTTP API microgateway that acts like a reverse proxy to capture, forward, measure, transform, filter and simulate API calls."